src/main/java/eu/darkbot/api/utils/NativeAction.java

```diff
@@ -32,17 +29,16 @@ public enum Mouse {
         }
 
         public long of(int x, int y) {
-            return toActionLong(message, (short) 1, (short) x, (short) y, false);
+            return toActionLong(message, (short) 1, x, false);
```

Comment:
Why do you remove the use of "y"?

Reply:
sorry my mistake i add:
 public long after(int x, int y) {
            return toActionLong(message, (short) 1, x, y, true);
        }